chesster.js

```diff
@@ -1263,7 +1263,6 @@ controller.on('ambient', function(bot, message) {
                 scheduling_reply_cant_schedule_others(bot, message);
                 return;
             }
-            console.log(message);
```

Comment:
That brings up a good question, I see this is being removed. But can we catch standard out and write to another log? 

I'll probably add this to the start script